### ISSUE TITLE
Gutter Blame avatar does not use Gravatar fallback style

### ISSUE THREAD

**Issue by Konsonanz** [eamodio/vscode-gitlens#1246]
#### Description:
As of GitLens Version 11+, the Gutter Blame fallback avatar does not use the Gravatar fallback style anymore, but rather the same blue-white icon for every commit. All other gitlens avatar displays (hover, sidemenu etc.) seem to still use the Gravatar fallback.
This is for me an inconsistency issue (between gutter and hover at least), as well as a comfort issue as I mostly work with an on premises git hosting solution which does not support user avatars and since now every author avatar uses one and the same fallback icon I can not visually distinguish different authors anymore.

#### Environment:
- GitLens Version: 11.0.6
- Git Version: 2.29.2
- VSCode Version: 1.52.0-insider (bc13785d3d) x64
- OS Version: Ubuntu 20.04.1 LTS

#### Steps to Reproduce:
1. Start Visual Studio Code with a new user profile (for testing, I ran `mv "~/.config/Code - Insiders" "~/.config/Code - InsidersBAK"`, prompting Visual Studio Code to create a new user profile on start)
2. Create a new test repository without a remote
```
$ mkdir /tmp/gitlens-test
$ cd /tmp/gitlens-test
$ git init
$ echo "test" > README.MD
$ git add -a
$ git commit -m "Initial commit"
```
3. Open the new repository and file and activate line blame:
![Bildschirmfoto von 2020-12-08 13-38-40](https://user-images.githubusercontent.com/46794176/101485097-9122a380-3952-11eb-9e74-eb05b360500c.png)

#### Expected Behaviour:
I expect the Gutter Blame avatar to use the same Gravatar fallback as the Hover Avatar.
#### Observed Behaviour
The Gutter Blame avatar shows a different fallback as the Hover Avatar and does not respect the `gitlens.defaultGravatarStyle` setting.

**Comment by eamodio** [comment]
Can you please verify this fix in today's insiders edition? Be sure to disable/uninstall the stable version of GitLens first.

You can install the insiders edition from [here](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens-insiders).

**Comment by Konsonanz** [comment]
Tested OK and can confirm as fixed with Gitlens (Insiders) Version: 2020.12.1005.

Thank you very much for the quick fix. Love the extension!

**Comment by eamodio** [comment]
Thank you for verifing!
